docs/concepts/trading-on-vega/orders.md

```diff
@@ -106,33 +107,55 @@ Unparked pegged orders will be rejected:
+* Peak size - Size of the order that is visible and be filled by a single order.
+* Minimum visible size - Amount the peak size needs to drop to before it is refreshed.
 
-A post-order cannot be active at the same time as a reduce-only order on the same market. 
+An iceberg order must have a persistent [time in force](#times-in-force): GTC, GTT, GFA, GFN.
 
-#### Reduce-only 
-Reduce-only is only an available option for orders with a non-persistent time-in-force. If set, the order will only be executed if the outcome of the trade moves the trader's position closer to 0. 
+The price for an iceberg order can be set like a standard limit order, or it can be pegged to a reference price.
 
-In addition, a reduce-only order will not move a position to the opposite side from the trader's current position. For example, if the trader's current position is a short, enabling reduce-only cannot make the trader long as a result. If submitted with an IOC time in force, where the full volume would switch sides, only the amount required to move the position to 0 will be executed.
+#### Iceberg order execution
+If an iceberg order, on entry, crosses with the best bid/ask price, it will trade using its full quantity, **not** the peak size. This prevents an iceberg order from being crossed after refreshing.
 
-A reduce-only order cannot be active at the same time as a post-only order on the same market. 
+When the iceberg order enters the order book passively, it trades like a typical persistent order, where the peak size is the displayed size.
+
+On refresh, the replenished order will get a new time priority, as it's treated like a new order.
+
+#### Amend an iceberg order
+An iceberg order can be amended. The remaining amount can be increased or decreased, while the peak size will stay the same. If the amended remaining amount is smaller than the peak size, the peak size is automatically reduced to be the same as the amended remaining amount.
+
+Amending an iceberg does not affect the peak size's time priority.
+
+### Stop order
+A stop order is an order to buy or sell once the price reaches a specified price, known as the trigger price. Stop orders can be used to help a trader limit losses (stop loss), or capitalise on a gain (take profit) automatically when they already have an open position.
+
+Stop orders can only be used to bring a trader's position closer to zero, and thus make use of the [reduce only](#reduce-only) condition and will only be accepted if the trader already has an open position on the market.
+
+A stop order must include a trigger price - for a take profit stop order, or percentage - for a trailing stop, and whether to deploy the order once the last traded price "rises above" or "falls below" the given stop price/percentage. It can, but doesn't need to, have an expiry date and time, or an execution time (???) at which point the order will be filled if possible.
```

Comment:
```suggestion
A stop order must include a trigger price at a fixed level, or it can be a percentage for a trailing stop. It must also include whether to deploy the order once the last traded price "rises above" or "falls below" the given stop price/percentage. It can, but doesn't need to, have an expiry date and time, or an execution time at which point the order will be filled, if possible.
```